libs/platform/experience/src/services/layout/plugins/layout-plugins.providers.ts

```diff
@@ -47,6 +47,13 @@ export const layoutPluginsProviders: Provider[] = [
         (m) => m.SpacingStylePlugin
       ),
   },
+  {
+    provide: LayoutStylesPlugin,
+    asyncClass: () =>
+      import('./styles/image/image-style.plugin').then(
```

Comment:
Maybe better to choose another name for this plugin? `ObjectsStylesPlugin` for example, because this styles can be potentially applied for any 'object-like' elements: iframes, video, ...